### PR TITLE
Add global (click) event listener

### PR DESCRIPTION
**Pull request by MariyaShavshy**
Using finder from https://github.com/antonmedv/finder

### CODE REVIEW

File: src/pages/Content/index.js [hunk 1]
```diff
@@ -1,3 +1,9 @@
+// import object selector to easily fetch DOM elements
+import { finder } from '@medv/finder'
+
+// Global recorded events array
+var recordedEvents = [];
```
Comment (agamm) [inline src/pages/Content/index.js:5]: Make sure to merge the last PR before this one so we won't have conflicting histories.

File: src/pages/Content/index.js [hunk 2]
```diff
@@ -26,3 +32,28 @@ function click(selector) {
...
 }
+
+// a function that listens to click events and stores them in the recordedEvents array
+function attachGlobalEventListeners() {
+
+  // capture all click events
+  document.body.addEventListener('click', function (e) {
+
+    // find the best selector for click target (id/class/tag/attr)
+    const selector = finder(e.target);
```
Comment (agamm) [inline src/pages/Content/index.js:43]: Did you check with event [bubbling/capturing](https://stackoverflow.com/questions/4616694/what-is-event-bubbling-and-capturing)?

File: src/pages/Content/index.js [hunk 3]
```diff
@@ -26,3 +32,28 @@ function click(selector) {
...
+
```
Comment (agamm) [inline src/pages/Content/index.js:44]: What if `selector` is not found?

File: src/pages/Content/index.js [hunk 4]
```diff
@@ -26,3 +32,28 @@ function click(selector) {
...
+    // store selector in recordedEvents
+    recordedEvents.push({
+      type: 'click',
+      element: selector,
+      time: new Date().getTime()
+    });
+
+    // test wether the event was added
```
Comment (agamm) [inline src/pages/Content/index.js:52]: :+1:

**Review by agamm — COMMENTED** [review]
Some comments

File: src/pages/Content/index.js (hunk 2 above)
Comment (MariyaShavshy) [reply src/pages/Content/index.js:43]: I can add `{capture: true}` and push the changes, I'm just not sure if it's necessarily better than bubbling.
Let me know

File: src/pages/Content/index.js (hunk 3 above)
Comment (MariyaShavshy) [reply src/pages/Content/index.js:44]: Added a try and catch clause, just waiting on confirmation on capturing before I push the changes

File: src/pages/Content/index.js (hunk 2 above)
Comment (agamm) [reply src/pages/Content/index.js:43]: No, just check that the click works well with nested components :+1:

Comment (agamm) [reply src/pages/Content/index.js:43]: @MariyaShavshy let me know when you're done.

Comment (MariyaShavshy) [reply src/pages/Content/index.js:43]: @agamm It works great, I've already checked it a while back in the `console.debug()` section...

File: src/pages/Content/index.js (hunk 3 above)
Comment (MariyaShavshy) [reply src/pages/Content/index.js]: I've just pushed the try catch block @agamm both here and in #20 so we don't get any conflicts when merged. I've verified that the click listener works great before making this pull request (please refer to the `console.debug()` test within the try block)

**Review by agamm — APPROVED** [review]
Looks good! awesome work!

**Pull request merged**